### DISCUSSION TITLE
Are we supposed to know what the "Up" arrows and numbers mean?

### DISCUSSION THREAD

**Discussion by gilgongo** [community/community#3657] [Discussions] [answered]
I'm surprised nobody has asked. They have no tooltips, no explanation. What do they do?

![Screenshot 2021-03-31 at 7 48 51 AM](https://user-images.githubusercontent.com/4561747/113102243-acf9d000-91f5-11eb-816c-ea8613d75431.png)

![bitmoji](https://sdk.bitmoji.com/render/panel/83c36bb9-c15e-44f7-9379-ba4c4bd4bb42-e453f381-f13d-466a-a20e-c03c786b333b-v1.png?transparent=1&palette=1&width=246)

## Possible interpretations

1. It serves as another way of "agreeing" with the post, together with the existing "thumbs up" emoji.
2. ~~If the post is at the top of the thread, it raises that thread in the stack position.~~
3. It's explained on a tooltip, but the tooltip isn't working. 
4. If enough people click it for any of the posts on a thread, the whole thread moves up the list.
5. ~~It does nothing and only you can see it (similar to a "starred" item).~~
6. It confers some kind of rank on the poster (although no such ranking is displayed on the user's profile).
7. ~~It does nothing.~~

**Comment by noah-nuebling** [comment]
My intuitive interpretation would be that it increases visibility, in contrast to the thumbs up emoji which is just a social thing without any “special effects”.

I think I’m subconsciously relating it to Reddit upvotes.

**Reply by gilgongo** [reply]
I see - so the “special effect" is that the more "ups" a parent post gets, the higher it rises in the stack? I can't work out what it does to comments on parents though, since it can't alter their order, yet all posts have the "up" (arrow and no "down").

I love how it's just there with no explanation, no easy interpretation, and so presumably most people ignore it as noise.

**Reply by noah-nuebling** [reply, marked as answer]
If you sort the comments by "top" it will order them by the number of upvotes too I think.

**Comment by hagaigold** [comment]
did you find the answer?

**Reply by gilgongo** [reply]
I think the closest we have is that it participates in sorting. So I'll mark that as the answer. Would it have killed them to put a tooltip on it saying "Used when viewing this discussion by top comments"? I dunno - designers these days...

**Reply by hagaigold** [reply]
probably the one who implemented it, already moved to his next job :)  
We always found ourselves as a new "monkey" once in a while -  https://www.proserveit.com/blog/five-monkeys-experiment-lessons

**Comment by sugeshc389** [comment]
Response #10285

**Comment by knoepfel** [comment]
See the answer here: https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#upvoting-a-discussion
